src/util/Constants.js

```diff
@@ -508,3 +508,49 @@ exports.Colors = {
   DARK_BUT_NOT_BLACK: 0x2C2F33,
   NOT_QUITE_BLACK: 0x23272A,
 };
+
+exports.APIErrors = {
```

Comment:
Should probably document this (probably just turn this object into a docs comment) so it turns up in the docs.